pwa-devdocs/package.json

```diff
@@ -8,15 +8,16 @@
         "jquery": "^3.4.1"
     },
     "scripts": {
-        "build": "npm-run-all -s build-js build-css build-reference-doc-snippets build-compatibility-table build-html",
-        "build-compatibility-table": "node scripts/create-compatibility-table.js",
-        "build-css": "sass --style=compressed src/_scss/main.scss:src/builds/css/main.css",
-        "build-html": "bundle exec jekyll build --source src -I",
-        "build-js": "webpack --config webpack.config.js",
-        "build-production": "npm-run-all -s clean build-js build-css && jekyll build --source src/ --config src/_config.yml,_config-production.yml",
-        "build-reference-doc-snippets": "node scripts/create-reference-docs",
+        "build": "npm-run-all -s build:js build:css build:reference-doc-snippets build:compatibility-table build:env-var-definition-doc build:html",
```

Comment:
Do these scripts have to run in series? If not, there's a way you can simplify this a lot.